app/assets/javascripts/app/chosen_search_options.js

```diff
@@ -1,10 +1,31 @@
-$(function () {
+const forceChosenInput = (select, data, query) => {
+  if (!select || !data || data.length) return;
+
+  const optionText = query
+    ? `No results found for: ${query}`
+    : "No results found";
+  select.append(
+    `<option value="" class="no-results-option"> ${optionText} </option>`
+  );
+
+  select.trigger("chosen:updated");
+};
+
+const csrfToken = (selectField) => {
+  const isTestEnv = selectField.data("testEnv");
+
+  if (isTestEnv) {
+    return "";
+  }
```

Comment:
why is this needed?

Reply:
I think on the test env it'll just follow the case where `csrfTokenElement` is null

Reply:
The problem I had is that when `csrfTokenElement` was null, the trigger wasn't loaded. At some point this was necessary but then I did a refactor and I think you're right